### DISCUSSION TITLE
Cannot Pass Uuuid from blade component to livewire component

### DISCUSSION THREAD

**Discussion by Soltee** [livewire/livewire#3888] [1. Help] [answered]
Greetings!

Tables have Uuid Instead of regular id. But when I pass the Uuid from the blade to livewire component, I cannot , instead causes js error.

Eg. 
  1. Blade
     <button 
            wire:click="delete({{$review->uuid}})"
            type="button"
            class="px-5 mt-2 py-1 rounded-lg bg-red-400 hover:bg-red-500 text-white" 
            >
              Delete
    </button>

 2. Component Livewire
    public function delete($id)
    {
        $review = Review::findOrfail($id);
        if ($review && ($review->customer_id == auth()->guard('customer')->user()->id)) {
            $review->delete();
        }
        $this->rated    = true;
    }

RESULTS.
When click on Delete Button Above, error says on console
- Uncaught SyntaxError: identifier starts immediately after numeric literal

**Comment by joshhanley** [comment, marked as answer]
@Soltee can you please share your blade code for your delete button?

Thanks!

**Reply by Soltee** [reply]
@joshhanley 
Here is my blade code for delete button
Note::The review id is an Uuid 

       <button 
               wire:click="delete({{$review->id}})"
               type="button"
               class="px-5 mt-2 py-1 rounded-lg bg-red-400 hover:bg-red-500 text-white" 
               >
                  Delete
        </button>

Thank you!!!

**Reply by joshhanley** [reply]
@Soltee because it’s a uuid, you just need to put singles quotes around it inside the parentheses, like this

```blade
   <button 
           wire:click="delete('{{$review->id}}')"
           type="button"
           class="px-5 mt-2 py-1 rounded-lg bg-red-400 hover:bg-red-500 text-white" 
           >
              Delete
    </button>
```

Hope this helps!

**Reply by Soltee** [reply]
@jacovdbergh
Hooray!! It worked man. 
Thank you for your time.
